README.md

```diff
@@ -1,222 +1,166 @@
+# Eightshift WordPress Boilerplate
 
 [![Travis](https://img.shields.io/travis/infinum/wp-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/wp-boilerplate)
 [![GitHub tag](https://img.shields.io/github/tag/infinum/wp-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/wp-boilerplate)
 [![GitHub stars](https://img.shields.io/github/stars/infinum/wp-boilerplate.svg?style=for-the-badge&label=Stars)](https://github.com/infinum/wp-boilerplate/)
 [![license](https://img.shields.io/github/license/infinum/wp-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/wp-boilerplate)
 
 
-
 This repository contains all the tools you need to start building a modern WordPress theme, using all the latest front end development tools.
 
-## Who do I talk to?
+## Getting started
 
-For questions talk to:
+First you need to install WordPress locally, using any of the local development environment you prefer. You can use XAMPP, MAMP, WAMP, VVV, Docker or Laravel Valet. You'll also need to have [Node.js](https://nodejs.org/en/), [Composer](https://getcomposer.org/) and [WP-CLI](https://wp-cli.org/) installed.
 
-* [ivan.ruzevic@infinum.hr](ivan.ruzevic@infinum.hr)
-* [denis.zoljom@infinum.hr](denis.zoljom@infinum.hr)
-* [team@eightshift.com](team@eightshift.com)
+## Initial Setup
+1. To start, fork this repository to your own, and then clone it. If you are using VVV clone it in the `public_html` folder:
 
-## It contains:
-
-* Webpack4+ config and build
-* ES Linting
-* Style Linting
-* PHP Error Check
-* WPCS Style Guide
-* BrowserSync
-* PostCSS with plugins
-* Build and Prebuild Bash Script
-* .gitignore
-* Sass files based on Infinum Handbook
-* Precreated template files and helpers
-* BEM menues
-* Google rich snippets
-* Object oriented codebase
-* Namespacing
-* Autoloader
-* Project-specific wp-config
-* Import / export scripts
-* Project setup wizard
-* ...
+    ```bash
+    git clone git@github.com:your-name/wp-boilerplate.git
+    ```
 
-## Getting started
+2. Install latest version of [Node.js](https://nodejs.org/en/)
 
-First you need to install WordPress locally, using any of the local development environment you prefer. You can use XAMPP, MAMP, WAMP, VVV, Docker or Laravel Valet. You'll also need to have [Node.js](https://nodejs.org/en/), [Composer](https://getcomposer.org/) and [WP-CLI](https://wp-cli.org/) installed.
+3. Install latest version of [Composer](https://getcomposer.org/)
 
-To start, fork this repository to your own, and then clone it
+4. Install latest version of [WP-CLI](https://wp-cli.org).
 
-```bash
-git clone git@github.com:your-name/wp-boilerplate.git
-```
+5. Run script to setup your project and rename all files via wizard. This will make changes to theme name, description, author, text domain, package, namespace, and constants (this is important when specifying environment variable):
+**Run this only once!**
 
-If you are using VVV clone it in the `public_html` folder
+    ```bash
```

Comment:
rename should be done either with shell script or using node script. In order for node script to work, `npm install` should be ran and then `npm run rename` iirc. This should be added to the readme.

Reply:
you run sh bin/rename.sh and that runs 
````
npm install
npm run rename
````

this shell script should run on windows and osx

Reply:
I'll try this out on Windows :+1: